### ISSUE TITLE
Alias and Import

### ISSUE THREAD

**Issue by Host32** [iraasta/autocomplete-elixir#6]
In my local project when i use 'import' and 'alias' with some module, the auto complete for they doesn't works

Ex:

```elixir
defmodule MyProject.MyModule do
   import MyProject.AnotherModule
   alias MyProject.AliasModule, as: FastAccess

   def my_func do
      FastAccess.func() # here
      func_by_another_module() # and here, the auto complete doesn't works
   end
end
```

**Comment by iraasta** [comment]
Yes. But that's a problem in Elixir itself. I can't do anything about it except for writing the whole thing myself 

Check this out 
https://github.com/elixir-lang/elixir/issues/2519

**Comment by Host32** [comment]
wow...

Ok, Thank you :)

**Comment by iraasta** [comment]
No problem :) 
Glad You're using my extension 

Cheers

**Comment by lexmag** [comment]
Hi everyone, the problem with aliases is already fixed there: https://github.com/elixir-lang/elixir/pull/3280. :bowtie:

**Comment by Mazyod** [comment]
Hmm.. even though I am running Elixir 1.2.2, with the alias stuff fixed and all, the autocomplete doesn't seem to work.
